hack/release.toml

```diff
@@ -76,6 +76,13 @@ Talos now supports Jetson Nano SBC.
         description="""\
 `talosctl` commands which accept JSON patches (`gen config`, `cluster create`, `patch machineconfig`) now support multiple patches, loading patches
 from files with `@file.json` syntax, and support loading from YAML format.
+"""
+
+    [notes.staticpods]
+        title = "Static Pods in the Machine Configuration"
+        description="""\
+Talos now accepts static pods definitions in the `.machine.pods` key of the machine configuration.
+Static pod definitions can be updated without a node reboot.
```

Comment:
May want to also specify the validation thing here so it's clear in release notes.